addons/dialog/index.jsx

```diff
@@ -6,7 +6,17 @@ module.exports = React.createClass({
 
   getDefaultProps() {
     return {
-      className: 'col-dialog'
+      className: 'col-dialog',
+      headingLevel: 'h3'
+    }
+  },
+
+  getTitle() {
+    let { headingLevel, title } = this.props
+    if (title) {
+      return React.DOM[headingLevel]({
```

Comment:
`createElement` can accept a string value for a known entity (like `h3`) or a component definition